app/views/items/_index.html.erb

```diff
@@ -18,14 +18,15 @@
       <% if item.active? %>
 
         <p>Active</p>
-        <%= link_to "Deactivate", "/merchant/items/#{item.id}/toggle_active", method: :patch if current_merchant_user? && current_employee?(@merchant.id) %>
+
+        <%= link_to "Deactivate", "/merchant/items/#{item.id}/toggle_active", method: :patch if @merchant && current_merchant_user? && current_employee?(@merchant.id) %>
```

Comment:
Nice catch on this. My next PR should also be fixing this issue, so we should keep an eye out for merged conflicts when I push my branch up